src/examples/zkapps/hello_world/hello_world.ts

```diff
@@ -17,17 +18,23 @@ export class HelloWorld extends SmartContract {
 
   deploy(input: DeployArgs) {
     super.deploy(input);
-    this.x.set(Field(3));
+    this.setPermissions({
+      ...Permissions.default(),
+      editState: Permissions.proofOrSignature(),
+    });
+    this.x.set(Field(2));
+
     Party.setValue(this.self.update.delegate, adminPublicKey);
   }
 
   @method update(squared: Field, admin: PrivateKey) {
     const x = this.x.get();
-    this.x.assertEquals(x);
+    this.x.assertEquals(this.x.get());
```

Comment:
We [decided](https://docs.google.com/document/d/1ha2pyPn7ksIcKRd2w2Oj9J-xFvAxoCw0DNsApVpic9s/edit?disco=AAAAcV5-Vd4) to remove this assertion.

Reply:
An error will be produced without this precondition with the current version of SnarkyJS.